src/main/java/org/apache/maven/plugins/jlink/JLinkMojo.java

```diff
@@ -368,6 +368,14 @@ public class JLinkMojo extends AbstractJLinkMojo {
     @Parameter
     private List<Resource> additionalResources;
 
+    /**
+     * Add prefix to all of zip entries. It would be treated as a directory if the prefix ends with "/".
```

Comment:
I think this comment needs clarification, as it goes into the docs (site).

* If it does not end with `/`, what will happen?
* Isn't the prefix always supposed to be a directory?
* What does the default do (explain to users)?

Reply:
> Isn't the prefix always supposed to be a directory?

Yes, I think so because if the prefix does not have "/" in its tail, it breaks JDK directory structure - it means any launcher and `java` command in `bin` directory would not work. I will fix, and clarify Javadoc.

Reply:
@bmarwell 
I fixed docs, and always treats this value as a directory. Could you review again?